tests/nn/test_mlp.py

```diff
@@ -12,8 +15,30 @@ def test_mlp_forward():
 def test_SwiGLU_forward():
     n_embd = 512
     bias = True
-    model = SwiGLU(n_embd, bias)
-    input_tensor = torch.randn(1, n_embd)
-    output_tensor = model(input_tensor)
-    assert output_tensor.shape == (1, n_embd)
+    mlp = SwiGLU(n_embd, bias)
+
+    hidden_dim = 1536
+    assert mlp._get_hidden_dim(n_embd) == hidden_dim
+    
+    n_embd = 511
+    assert mlp._get_hidden_dim(n_embd) == hidden_dim

```

Comment:
if we make it static: 

```suggestion
    assert SwiGLU._get_hidden_dim(n_embd) == hidden_dim

```

Reply:
addressed in https://github.com/Modalities/modalities/pull/154/commits/4e579b6026b4b0d577892bf68d946af1f8aa545e